lib/plugins/openaps.js

```diff
@@ -39,6 +39,10 @@ function init (ctx) {
       , warn: sbx.extendedSettings.warn ? sbx.extendedSettings.warn : 30
       , urgent: sbx.extendedSettings.urgent ? sbx.extendedSettings.urgent : 60
       , enableAlerts: sbx.extendedSettings.enableAlerts
+      , predIOB: sbx.extendedSettings.predIob ? sbx.extendedSettings.predIob : '#ff00ff'
```

Comment:
Naming: perhaps `predIOBColor`  instead of `predIOB` to clearly indicate it's just about the color